soroban-sdk/src/symbol.rs

```diff
@@ -0,0 +1,29 @@
+#[cfg(doc)]
+use crate::Symbol;
+
+/// Create a [Symbol] with the given string.
+///
+/// The [Symbol] is generated at compile time and returned as a const.
+///
+/// ### Examples
+///
+/// ```
+/// use soroban_sdk::{sym, Symbol};
+///
+/// let symbol = sym!("a_str");
+/// assert_eq!(symbol, Symbol::from_str("a_str"));
+/// ```
+///
+/// ```
+/// use soroban_sdk::{sym, Symbol};
+///
+/// const symbol: Symbol = sym!("a_str");
+/// assert_eq!(symbol, Symbol::from_str("a_str"));
+/// ```
+#[macro_export]
+macro_rules! sym {
```

Comment:
Even if it is a few more characters it might improve readability if this macro was `symbol!`. Wdyt @jonjove @sisuresh @paulbellamy?

Reply:
I updated the change in 8836979d to use `symbol!`.

Reply:
+1, I would have chosen `symbol!` as well.